--- hatari-1.9.0/src/CMakeLists.txt.0000~	2015-09-10 23:16:03.000000000 +0200
+++ hatari-1.9.0/src/CMakeLists.txt	2016-10-08 01:18:25.465427420 +0200
@@ -12,7 +12,7 @@ set(SOURCES
 	video.c wavFormat.c xbios.c ymFormat.c)
 
 # Disk image code is shared with the hmsa tool, so we put it into a library:
-add_library(Floppy createBlankImage.c dim.c msa.c st.c zip.c)
+add_library(Floppy STATIC createBlankImage.c dim.c msa.c st.c zip.c)
 
 # When building for OSX, define specific sources for gui and ressources
 if(ENABLE_OSX_BUNDLE)
--- hatari-1.9.0/src/debug/CMakeLists.txt.0000~	2015-09-10 23:16:02.000000000 +0200
+++ hatari-1.9.0/src/debug/CMakeLists.txt	2016-10-08 01:18:25.465427420 +0200
@@ -7,7 +7,7 @@ if(ENABLE_DSP_EMU)
 	set(DSPDBG_C debugdsp.c)
 endif(ENABLE_DSP_EMU)
 
-add_library(Debug
+add_library(Debug STATIC
 	    log.c debugui.c breakcond.c debugcpu.c debugInfo.c
 	    ${DSPDBG_C} evaluate.c history.c symbols.c
 	    profile.c profilecpu.c profiledsp.c
--- hatari-1.9.0/src/falcon/CMakeLists.txt.0000~	2015-09-10 23:16:03.000000000 +0200
+++ hatari-1.9.0/src/falcon/CMakeLists.txt	2016-10-08 01:18:25.465427420 +0200
@@ -7,7 +7,7 @@ if(ENABLE_DSP_EMU)
 	set(DSP_SOURCES dsp_core.c dsp_cpu.c dsp_disasm.c)
 endif(ENABLE_DSP_EMU)
 
-add_library(Falcon
+add_library(Falcon STATIC
 	crossbar.c dsp.c ${DSP_SOURCES}
 	hostscreen.c microphone.c nvram.c videl.c
 	)
--- hatari-1.9.0/src/gui-sdl/CMakeLists.txt.0000~	2016-10-08 01:18:25.465427420 +0200
+++ hatari-1.9.0/src/gui-sdl/CMakeLists.txt	2016-10-08 01:18:33.153308063 +0200
@@ -5,7 +5,7 @@ if(CMAKE_COMPILER_IS_GNUCC)
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-write-strings")
 endif(CMAKE_COMPILER_IS_GNUCC)
 
-add_library(GuiSdl
+add_library(GuiSdl STATIC
 	dlgAbout.c dlgHalt.c dlgAlert.c dlgDevice.c dlgFileSelect.c dlgFloppy.c
 	dlgHardDisk.c dlgJoystick.c dlgKeyboard.c dlgMain.c dlgMemory.c
 	dlgNewDisk.c dlgRom.c dlgScreen.c dlgSound.c dlgSystem.c
--- hatari-1.9.0/src/uae-cpu/CMakeLists.txt.0000~	2015-09-10 23:16:03.000000000 +0200
+++ hatari-1.9.0/src/uae-cpu/CMakeLists.txt	2016-10-08 01:18:25.465427420 +0200
@@ -52,7 +52,7 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-variable -Wno-unused-label")
 endif(CMAKE_COMPILER_IS_GNUCC)
 
-add_library(UaeCpu
+add_library(UaeCpu STATIC
 	cpudefs.c cpuemu.c cpustbl.c
 	hatari-glue.c memory.c newcpu.c readcpu.c fpp.c
 	)
